templates/base.html

```diff
@@ -1,11 +1,13 @@
 <!doctype html>
 <html>
 <head>
-    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
-    <title>{% block title %}Recepies Blog{% endblock %}</title>
+    {% block head %}
```

Comment:
What does this `block head` give you exactly?

Reply:
Block head contains a link pointing to the bootstrap cdn . 
Just a file required by bootstrap !